package.json

```diff
@@ -11,8 +11,9 @@
     "db-seed": "NODE_ENV=testing sequelize db:seed:all",
     "test": "NODE_ENV=testing npm run dropTables && NODE_ENV=testing npm run db-migrate && npm run db-seed && NODE_ENV=testing nyc --reporter=text --reporter=html --require @babel/polyfill --require @babel/register mocha ./src/tests/*  --timeout 200000 --exit",
     "coverage": "nyc npm test  && npm run coveralls-coverage",
-    "dev-start": "nodemon --exec babel-node ./src/index.js",
+    "dev-start": "npx sequelize-cli db:seed:all && nodemon --exec babel-node ./src/index.js",
```

Comment:
It is not good practice to seed data every time you start development server for example `npx sequelize-cli db:seed:all` 

I recomment to seprate seed scripts for dev and test  and call it when you need.
```
"db-seed": "NODE_ENV=testing npx sequelize-cli db:seed:undo:all && NODE_ENV=testing sequelize db:seed:all",
"db-seed-dev": "npx sequelize-cli db:seed:undo:all && sequelize db:seed:all",
```